src/lib/auth/userDetails.ts

```diff
@@ -1,16 +1,24 @@
 import { jwtDecode } from "jwt-decode";
 import { UserDetailsReturnProps } from "../../types/auth/userDetails";
-import { getTokenSilently, getAuth0Client } from "./auth0";
 import { DecodedJWT } from "../../types/auth/login";
 
 export async function userDetails(): Promise<UserDetailsReturnProps> {
-  const auth0 = await getAuth0Client();
-  const authParams = {
-    transaction_input: JSON.stringify({ othentFunction: "idToken" }),
-  };
-  const accessToken = await getTokenSilently(auth0, authParams);
-  const JWT = accessToken.id_token;
-  const decoded_JWT: DecodedJWT = jwtDecode(JWT);
+  const id_token = localStorage.getItem("id_token");
+
+  if (!id_token) {
+    const userDetailsString = localStorage.getItem("userDetails");
+    if (userDetailsString) {
+      return JSON.parse(userDetailsString) as UserDetailsReturnProps;
+    } else {
+      throw new Error("Error retrieving createUser details.");
```

Comment:
Is the error message correct?

Reply:
No! Good catch thanks :)

Reply:
utACK lol nice https://github.com/bitcoin/bitcoin/issues/6100, I like the lingo